plugins/discourse-local-dates/assets/javascripts/initializers/discourse-local-dates.js

```diff
@@ -238,39 +238,37 @@ export default {
       return;
     }
 
-    const siteSettings = owner.lookup("site-settings:main");
-    if (event?.target?.classList?.contains("discourse-local-date")) {
-      if ($(document.getElementById("d-popover"))[0]) {
-        hidePopover(event);
-      } else {
-        showPopover(event, {
-          htmlContent: buildHtmlPreview(event.target, siteSettings),
-        });
-      }
-    } else if (event?.target?.classList?.contains("download-calendar")) {
+    if (event?.target?.classList?.contains("download-calendar")) {
       const dataset = event.target.dataset;
-      hidePopover(event);
       downloadCalendar(dataset.title, [
         {
           startsAt: dataset.startsAt,
           endsAt: dataset.endsAt,
         },
       ]);
-    } else {
-      hidePopover(event);
+      return;
+    }
+
+    if (!event?.target?.classList?.contains("discourse-local-date")) {
+      return;
     }
+
+    const siteSettings = owner.lookup("site-settings:main");
+
+    showPopover(event, {
```

Comment:
It would be great if we could document the `showPopover` helper in the PR description, i.e. a click sample and a hover sample.

Or we could add a styleguide page?

Reply:
This is tippy API exactly. I can add a link to it if you want.

Reply:
Added a comment with a  link